packages/loader/src/gltf/parser/GLTFParserContext.ts

```diff
@@ -87,11 +85,16 @@ export class GLTFParserContext {
               : Promise.all<T>(glTFItems.map((_, index) => this.get<T>(type, index)));
         } else {
           resource = parser.parse(this, index);
-          this._handleSubAsset(resource, type, index);
         }
       } else {
         resource = Promise.resolve<T>(null);
       }
+    } else {
+      resource = parser.parse(this, index);
+    }
+
+    if (index !== undefined || !glTFKey) {
+      this._handleSubAsset(resource, type, index);
```

Comment:
`GLTFParserType.Schema` and `GLTFParserType.Validator` will called

Reply:
yes, i want discard it in `_handleSubAsset`.
or i write `if (index !== undefined || ( glTFResourceMap[type] && !glTFKey) {` here